src/utils/helperfunctions.ts

```diff
@@ -91,13 +91,21 @@ export function NodeIDStringToBuffer(pk: string): Buffer {
 export function costImportTx(tx: UnsignedTx): number {
   let cost: number = calcBytesCost(tx.toBuffer().byteLength)
   const importTx = tx.getTransaction() as ImportTx
-  importTx.getImportInputs().forEach((input: TransferableInput) => {
-    const inCost = input.getCost()
+  importTx.getImportInputs().forEach((input: TransferableInput): void => {
+    const inCost: number = input.getCost()
     cost += inCost
   })
   return cost
 }
 
 export function calcBytesCost(len: number): number {
-  return len * Defaults.network[1].C["txBytesGas"]
+  return len * Defaults.network[1].C.txBytesGas
+}
+
+export function costExportTx(tx: UnsignedTx): number {
+  let bytesCost: number = calcBytesCost(tx.toBuffer().byteLength)
+  const exportTx = tx.getTransaction() as ExportTx
+  const numSigs: number = exportTx.getInputs().length
+  const sigCost: number = numSigs * Defaults.network[1].C.costPerSignature
+  return bytesCost + sigCost
```

Comment:
This seems to be missing the fixed fee of 10,000 gas per atomic transaction that we are adding in AP5 (https://github.com/ava-labs/avalanche-docs/pull/405/files)

Reply:
This might explain the need to subtract one avax from the balance to get things to work correctly

Reply:
Resolved